robot/EDA/tests/browser/settings/system.robot

```diff
@@ -4,15 +4,31 @@ Resource        robot/EDA/resources/EDA.robot
 Library         cumulusci.robotframework.PageObjects
 ...             robot/EDA/resources/SystemSettingsPageObject.py
 
-Suite Setup     Open Test Browser
+Suite Setup     Run keywords
+...             Initialize test data
+...             Open Test Browser
 Suite Teardown  Capture screenshot and delete records and close browser
 
+
+*** Keywords ***
+Initialize test data
+    [Documentation]                     Creates collaboration group and gets the SFID of group and
+    ...                                 also reads SFID of security user
+    ${group_name} =                     Set variable            Test Chatter Group
+    ${user_name} =                      Set variable            Security User
+    ${group_id}=                        API Create Collaboration Group  ${group_name}
+    ${user_id} =                        API Get ID          User     Name       Security User
+    Set Suite Variable  ${group_name}
+    Set Suite Variable  ${user_name}
+    Set Suite Variable  ${group_id}
+    Set Suite Variable  ${user_id}
```

Comment:
@rhakeshj Very minor suggestion maybe align all the variables so the indent is the same. For example going from one set to the next you can do:
```
    ${user_id} =                        API Get ID          User     Name       Security User
    Set Suite Variable                  ${group_name}
```
This just looks a little cleaner.